frappe/commands/site.py

```diff
@@ -340,6 +352,11 @@ def restore_backup(
 	# Validate the sql file
 	validate_database_sql(sql_file_path, _raise=not force)
 
+	if skip_definer:
+		err, out = frappe.utils.execute_in_shell(
+			f" sed 's/\\sDEFINER=`[^`]*`@`[^`]*`//g' -i {sql_file_path}", check_exit_code=True
+		)
```

Comment:
Backups can be gzipped, this wouldn't work there.

What's the reason for removing this anyway?